go.mod

```diff
@@ -139,7 +139,7 @@ require (
 	github.com/containerd/ttrpc v1.2.2 // indirect
 	github.com/coredns/caddy v1.1.1 // indirect
 	github.com/coreos/go-semver v0.3.1 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
```

Comment:
is it intentional this is pulling in a version between tags? do we know if this library expects to be used on a non-tagged version and is stable there?

Reply:
heh, ok